### ISSUE TITLE
[Bug]:   Minibreak countdown display not accurate

### ISSUE THREAD

**Issue by e-t-l** [hovancik/stretchly#956]
### Version

- [x] I'm using version 1.7.0

### Known issues

- [X] I've checked Known issues

### Existing issues

- [X] I've checked Existing issues

### What operating system are you using?

Windows

### Operating System Version

Windows 10 v 20H2

### Reproduction steps

1. Enable minibreaks with 20 second duration
2. Use computer (nothing too resource-intensive) until minibreak triggers
3. Observe
...
**Reproduces how often:** 100% of the time

### Expected Behavior

On-screen timer counts down from 20, and after counting to 1, disappears. Progress bar underneath countdown timer decreases proportionally from full to empty.

### Actual Behavior

The countdown timer starts around 15 and counts down to 1, then stays on 1 for several seconds before disappearing. Likewise, the progress bar starts around 3/4 full and decreases to empty when the countdown hits 1, then stays at empty for several seconds. It seems like the countdown is somehow starting early (or like the time the minibreak window is active is out of sync with it). I'm not certain whether the actual duration of the minibreak window being up is 20 seconds. 

### Relevant log output

```shell
[2021-05-09 22:16:17.276] [info]  Stretchly: showing window 1 of 1
[2021-05-09 22:16:17.280] [info]  Stretchly: starting Mini Break
[2021-05-09 22:16:18.492] [info]  Stretchly: finishing Mini Break
[2021-05-09 22:36:26.073] [info]  Stretchly: showing window 1 of 1
[2021-05-09 22:36:26.077] [info]  Stretchly: starting Mini Break
[2021-05-09 22:36:28.344] [info]  Stretchly: finishing Mini Break
[2021-05-09 22:56:35.284] [info]  Stretchly: showing window 1 of 1
[2021-05-09 22:56:35.287] [info]  Stretchly: starting Mini Break
[2021-05-09 22:56:55.319] [info]  Stretchly: finishing Mini Break
```

### Preferences

```shell
{
	"microbreakDuration": 20000,
	"microbreakInterval": 1200000,
	"breakDuration": 60000,
	"breakInterval": 1,
	"breakNotification": false,
	"microbreakNotification": false,
	"breakNotificationInterval": 30000,
	"microbreakNotificationInterval": 10000,
	"microbreak": true,
	"break": false,
	"microbreakStrictMode": false,
	"breakStrictMode": false,
	"morningHour": 6,
	"microbreakPostpone": false,
	"breakPostpone": true,
	"microbreakPostponeTime": 120000,
	"breakPostponeTime": 300000,
	"microbreakPostponesLimit": 1,
	"microbreakPostponableDurationPercent": 30,
	"breakPostponesLimit": 1,
	"breakPostponableDurationPercent": 30,
	"mainColor": "#1D1F21",
	"transparentMode": true,
	"opacity": 0.9,
	"audio": "silence",
	"volume": 1,
	"fullscreen": false,
	"ideas": false,
	"naturalBreaks": false,
	"naturalBreaksInactivityResetTime": 300000,
	"allScreens": true,
	"useIdeasFromSettings": false,
	"language": "en",
	"notifyNewVersion": true,
	"isFirstRun": false,
	"posLatitude": 0,
	"posLongitude": 0,
	"useMonochromeTrayIcon": true,
	"useMonochromeInvertedTrayIcon": false,
	"silentNotifications": true,
	"monitorDnd": true,
	"microbreakStartSoundPlaying": false,
	"breakStartSoundPlaying": false,
	"themeSource": "system",
	"endBreakShortcut": "CmdOrCtrl+X",
	"breakWindowWidth": 0.85,
	"breakWindowHeight": 0.85,
	"checkNewVersion": true,

}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hovancik** [comment]
Hi, I've seen this previously with my Linux station when my PC was in heavy usage. But when you say nothing resource heavy, I guess it must be the fact that I am using Electron, which is not very efficient. The break window is really simple, so not sure if that could cause the issue. 

Did you start to notice this with 1.7 or also before? 

I don't have Windows machine, but maybe someone else can chime in?

**Comment by hovancik** [comment]
Also: just noticed that you write Mini Break. Does it mean that Long Breaks are fine?

**Comment by e-t-l** [comment]
>Did you start to notice this with 1.7 or also before?

I've noticed this bug with all previous versions of Stretchly that I've used. I've always had my minibreaks set to 24 seconds, because that does get the timer to start at ~20 by the time the break window actually pops up. It's only with v1.7 that I've noticed that the countdown and progress bar "hang" for several seconds after reaching zero/empty. So I'm not 100% sure if the duration that the break window is visible is closer to 20 or 24 seconds, all things considered. (It's also possible that it was doing this behavior prior to v1.7 and I just hadn't noticed. Or possibly the behavior was present, but less pronounced.)

>Does it mean that Long Breaks are fine?

Personally, I like Stretchly to emulate a [20/20/20](https://www.healthline.com/health/eye-health/20-20-20-rule) system, so I actually have Long Breaks disabled. I just have 24-second mini breaks that fire every 20 minutes.

**Comment by hovancik** [comment]
could you try what happens with Long Breaks? Just trying to see if there's any difference. 

I've installed Stretchly in Virtualbox Windows machine but not having those issues.

**Comment by e-t-l** [comment]
Confirming that I see this behavior with Long Breaks as well.

**Comment by hovancik** [comment]
I think it also might be GPU thing as when I run Stretchly from source in console, it shows GPU is used. Is it possible to check in Windows GPU usage?

**Comment by hovancik** [comment]
I think I've fixed this in #978, we'll see with next release.
